application/actions/meeting_test.go

```diff
@@ -364,3 +371,73 @@ func (as *ActionSuite) Test_CreateMeetingInvites() {
 		}
 	}
 }
+
+func (as *ActionSuite) Test_RemoveMeetingInvite() {
+	f := createFixturesForMeetings(as)
+
+	var resp meetingInvitesResponse
+
+	const queryTemplate = `mutation { meetingInvites : removeMeetingInvite(input: %s) { %s } }`
+
+	type testCase struct {
+		Name           string
+		Email          string
+		MeetingID      string
+		TestUser       models.User
+		ResponseEmails []string
+		ExpectError    string
+	}
+
+	testCases := []testCase{
+		{
+			Name:           "bad email",
+			Email:          "not_invited@example.com",
+			MeetingID:      f.Meetings[0].UUID.String(),
+			TestUser:       f.Users[0],
+			ResponseEmails: []string{},
+			ExpectError:    "problem removing the meeting invite",
+		},
+		{
+			Name:           "not creator, not organizer",
+			Email:          "invitee2@example.com",
+			MeetingID:      f.Meetings[1].UUID.String(),
+			TestUser:       f.Users[1],
+			ResponseEmails: []string{},
+			ExpectError:    "not allowed",
+		},
+		{
+			Name:           "creator",
+			Email:          "invitee0@example.com",
+			MeetingID:      f.Meetings[2].UUID.String(),
+			TestUser:       f.Users[0],
+			ResponseEmails: []string{"invitee1@example.com"},
+		},
+		{
+			Name:           "organizer",
+			Email:          "invitee1@example.com",
+			MeetingID:      f.Meetings[2].UUID.String(),
+			TestUser:       f.Users[0],
+			ResponseEmails: []string{},
+		},
+	}
+
+	for _, tc := range testCases {
```

Comment:
I just noticed that this and some others aren't using the `t.Run(tc.name, func(t *testing.T) { ...})`  way of running the tests. 
I don't know if it's worth changing it.

Reply:
Good suggestion. I'll see what it would take to make that change.